### PR TITLE
[13.0][MIG] onchange_helper

### PR DESCRIPTION
**Pull request by Laurent-Corron**

### CODE REVIEW

File: onchange_helper/models/base.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        # we take the value of the field
+        # If self is an empty record we will have an empty value
+        if self:
+            self.ensure_one()
+            record_values = self._convert_to_write(
+                {
+                    field_name: self[field_name]
+                    for field_name, field in self._fields.items()
+                }
+            )
```
Comment (lmignon) [inline onchange_helper/models/base.py:48]: IMO, we should adapt the code to at least don't extract computed field on existing record.

Comment (rousseldenis) [reply onchange_helper/models/base.py:48]: @Laurent-Corron @lmignon Is this still relevant ?

**Review by rousseldenis — APPROVED** [review]
Code and functional review

**Comment by rousseldenis** [comment]
@pedrobaeza 

This migration is immediate (see migration commit). Functionnaly tested

**Comment by pedrobaeza** [comment]
Well, I'm not a very active user of this module, and I see one comment still unanswered. Better to close first that before merging.

And luckily, this can disappear on next version!

File: onchange_helper/models/base.py (hunk 1 above)
Comment (rousseldenis) [reply onchange_helper/models/base.py:48]: @lmignon @Laurent-Corron

Comment (lmignon) [reply onchange_helper/models/base.py:48]: We can leave this code as is..

**Review by lmignon — APPROVED** [review]

**Comment by rousseldenis** [comment]
@pedrobaeza Done!

**Pull request merged**